linode/token/framework_resource_schema.go

```diff
@@ -74,15 +75,15 @@ var frameworkResourceSchema = schema.Schema{
 					RequireReplacementWhenScopesChangedDescription,
 				),
 			},
-			CustomType: customtypes.RFC3339TimeStringType{},
+			CustomType: timetypes.RFC3339Type{},
```

Comment:
One more small change, expiry will also need to be computed since the API populates it with the date `2999-12-12T05:00:00Z` if null.